test/Microsoft.ML.Tests/Scenarios/BinaryClassification.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Trainers;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.ML.Scenarios
+{
+    public partial class ScenariosTests
+    {
+        public class BreastCancerData
+        {
+            [Column(ordinal: "0")]
+            public float Label;
+            [Column(ordinal: "1-9")]
+            [VectorType(9)]
+            public float[] Features;
+        }
+
+        public class BreastCancerPrediction
+        {
+            [ColumnName("PredictedLabel")]
+            public DvBool Cancerous;
+        }
+
+        [Fact]
+        public void SaveModelToOnnxTest()
+        {
+            string dataPath = GetDataPath(@"breast-cancer.txt");
+            var pipeline = new LearningPipeline();
+
+            pipeline.Add(new Data.TextLoader(dataPath)
+            {
+                Arguments = new TextLoaderArguments
```

Comment:
>TextLoaderArguments [](start = 32, length = 19)

This is quite odd. Why are the "arguments" in this arguments object, rather than just being on the text loader directly? As near as I can see, generally that is how things are arranged through the generated APIs.

Reply:
Not related to this change, please open a separate issue for this. 

---
In reply to: [191499134](https://github.com/dotnet/machinelearning/pull/248#discussion_r191499134) [](ancestors = 191499134)

Reply:
Sorry, I was actually asking a question. Is there no reason? Was it merely a mistake? I'm guessing yes and have opened #261, maybe you can share your thoughts there.

---
In reply to: [191521847](https://github.com/dotnet/machinelearning/pull/248#discussion_r191521847) [](ancestors = 191521847,191499134)

Reply:
Thanks for opening the issue, I'll discuss this over there. I agree we should move the Arguments to the entrypoints class if possible because it will make the code that uses entrypoints cleaner.